README.rst

```diff
@@ -237,7 +237,7 @@ shipped by
 `OpenWISP Monitoring <https://github.com/openwisp/openwisp-monitoring>`_
 but can be used to include any custom element in the dashboard.
 
-**Note**: templates are loaded before charts.
+**Note**: you can configure to load templates before or after charts.
```

Comment:
it is possible to register templates to be loaded befor or after charts using the ``after_charts``  keyword argument (see below).